### DISCUSSION TITLE
ssh-copy-id authenticating with public key fails then auth by password failed

### DISCUSSION THREAD

**Discussion by jhjames3** [blinksh/blink#1842] [Support]
### Configuration
blink version 16.2.1.826
iPad os 16.6
iPad pro 12.9 3rd gen

### Describe the bug

Context: 
new Raspap running on new RaspberryPi 4 4GB
Can connect to RP over ssh from MacBook Pro 
(Used Raspberry Pi Imager to set up RP setting user id, password,  host name and copy of ssh-id)
Tried to connect iPad via blink: ssh-copy-id id-rsa user@IP
Connected to IP
Server host key xxx
Attempting to authentication as 'user'
Attempting authentication with publickey
Authentication failed
Attempting interactive authentication
auth by password failed
prompted for password interactively:
I typed in the correct password (reset to known password on RP via ssh link from Mac book pro using 'sudo passed")
That too failed

expected one of those methods would have worked

**Comment by carloscabanero** [comment]
Hi! Can you try connecting through `ssh` first, and sending us the `ssh -vvv` output in case that fails?
